Gruntfile.js

```diff
@@ -62,6 +62,15 @@ module.exports = function (grunt) {
       }
     },
 
+    uglify: {
+      my_target: {
+        files: [{
+          src: ['sass/base/components/{,**/}*.js'],
```

Comment:
Would be cool to implement that 'ignore underscore filenames' so we can rename and import plugins within the component files themselves only to make sure stuff is imported in the right order. 

As it currently stands, everything is imported and concatenated alphabetically which isn't ideal.. @gusliedke, what do you think?

https://stackoverflow.com/questions/21166088/grunt-uglify-ignore-specific-files-folders